src/SFA.DAS.FAT.Web/Models/BreadCrumbs/IBreadCrumbs.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace SFA.DAS.FAT.Web.Models.BreadCrumbs;
+
+public interface IBreadCrumbs
+{
+    string HomeUrl { get; set; }
+    bool ShowHomeCrumb { get; }
+
+    string ApprenticeTrainingCoursesUrl { get; set; }
+    bool ShowApprenticeTrainingCoursesCrumb { get; }
```

Comment:
All the booleans can have default implementation here

Reply:
they do, in the concrete class.  You can't have logic in an interface.